docs/learning/frontend/Extensions.md

```diff
@@ -0,0 +1,202 @@
+# iTwin.js Extensions
+
+An iTwin.js Extension is a separate JavaScript module that can load on demand into an iTwin.js frontend application.
+The separate deliverable enables Extensions to provide extensibility of an iTwin.js application at runtime.
```

Comment:
instead of runtime what do you think of, "extensibility of an iTwin.js application without having to rebuild or re-deploy the application."?